openapi-cli/src/test/java/io/ballerina/openapi/generators/client/AllOfResponsesTests.java

```diff
@@ -41,7 +41,7 @@
     @Test(description = "Tests for returnType")
     public void getReturnTypeTests() throws IOException, BallerinaOpenApiException {
         OpenAPI response = getOpenAPI(RES_DIR.resolve("swagger/return_type/response_with_allof_reference.yaml"));
-        BallerinaSchemaGenerator ballerinaSchemaGenerator = new BallerinaSchemaGenerator(response);
+        BallerinaSchemaGenerator ballerinaSchemaGenerator = new BallerinaSchemaGenerator(response, false);
```

Comment:
do we need to add `false` there, since we have a constructor `BallerinaSchemaGenerator(OpenAPI openAPI)` by adding `nullable = false` ?

Reply:
yes, that is unnecessary.